reverse_proxy_handler.cc

```diff
@@ -175,16 +175,16 @@ std::string ReverseProxyHandler::sendRequestToOrigin(Request request, std::strin
           return "RequestHandler::Error";
       }
 
-      std::cerr << "Got past connecting to remote_host!" << std::endl;
+      // std::cerr << "Got past connecting to remote_host!" << std::endl;
 
       std::string remote_request = request.raw_request();
       size_t host_pos = remote_request.find("Host");
       size_t host_end = remote_request.find("\r\n", host_pos);
       size_t host_len = host_end - host_pos;
       remote_request.replace(host_pos, host_len, "Host: " + remote_host);
-      std::cerr << "---------Sending remote_request-----------\n"
-                << remote_request
-                << "----------End of remote_request----------\n";
+      // std::cerr << "---------Sending remote_request-----------\n"
```

Comment:
This is an example of where having a full logging library with adjustable verbosity levels would have made things very convenient.